virttest/utils_test/libvirt.py

```diff
@@ -3543,6 +3543,7 @@ def modify_vm_iface(vm_name, oper, iface_dict, index=0, virsh_instance=virsh):
     if iface_link:
         iface.link_state = iface_link
     if iface_source:
+        iface.del_source()
```

Comment:
I suggest to have separate param like del_source with boolean to check if need to del, then using
```
if del_source:
    iface.del_source()
```

Reply:
Actually, after second thought, I decide to close this pr. Thank u for the review ~